lib/league_statistics.rb

```diff
@@ -103,4 +103,144 @@ def worst_defense
       worst_defense == team_id
     end[worst_defense].team_name
   end
+
+  def highest_scoring_visitor
+    games_by_team = @game_teams.group_by do |game_team|
+      game_team.team_id
+    end
```

Comment:
Looks like we do this:
```
games_by_team = @game_teams.group_by do |game_team|
   game_team.team_id
end
```
9 different times so we could refactor by moving it out to a helper method.

Reply:
I agree! I am going to hold off on this change until we are through interation 5 as discussed earlier.